module/models/actor-group.mjs

```diff
@@ -197,30 +197,48 @@ export default class CrucibleGroupActor extends foundry.abstract.TypeDataModel {
    * Award milestones to the members of this group.
    * @param {number} quantity
    * @param {string[]} recipientIds
+   * @param {boolean} [createMessage=true]
    * @returns {Promise<void>}
    */
-  async awardMilestones(quantity, recipientIds) {
+  async awardMilestones(quantity, recipientIds, createMessage=true) {
```

Comment:
Let's put `recipientIds` and `createMessage` within an options object so there's room to grow the signature here:
```suggestion
  async awardMilestones(quantity, {createMessage=true, recipientIds}={}) {
```